domain/src/main/java/org/oppia/android/domain/platformparameter/syncup/PlatformParameterSyncUpWorker.kt

```diff
@@ -51,35 +53,45 @@ class PlatformParameterSyncUpWorker private constructor(
     }
   }
 
-  // Parses a map of platform parameter values into a [List<PlatformParameter>]. If the parameters
-  // are not of type String, Int or Boolean this function fails with an [IllegalArgumentException].
+  /**
+   * Parses a map of platform parameter values into a [List<PlatformParameter>]. If the parameters
+   * are not of type String, Int or Boolean this function fails with an [IllegalArgumentException].
```

Comment:
```suggestion
   * Parses a map of platform parameter values into a [List<PlatformParameter>]. Parameters must be
   * of type String, Int or Boolean.
```

We actually don't want to indicate the type of exception in this case since it's a runtime exception & not meant to be caught.

Reply:
Understood. Done